azure-pipelines.yml

```diff
@@ -95,4 +95,5 @@ steps:
   failOnStderr: false
   condition: and(always(), eq(variables['Build.SourceBranch'], 'refs/heads/development'))
   name: helmPush
-  displayName: 'Push helm chart to acr'
\ No newline at end of file
+  displayName: 'Push helm chart to acr'
+  
```

Comment:
I think you might have ended up with an extra trailing line here and it looks like git/GitHub still isn't happy for some reason... I wonder if it will be happy with a single newline at the end of line 98?

Would be good to fix but obviously no point spending much time on it for such a minor thing.